__tests__/projects.js

```diff
@@ -0,0 +1,165 @@
+import nock from 'nock'; // https://github.com/nock/nock
+
+import { Octokit } from '@octokit/core';
+import { paginateGraphql } from '@octokit/plugin-paginate-graphql';
+import { RepositoryProjectsManager } from '../projects.js'; // eslint-disable-line import/extensions
+
+const GraphQlOctokit = Octokit.plugin(paginateGraphql);
+const octokit = new GraphQlOctokit({ auth: 'fake-token-value' }); // dont' use default GITHUB_TOKEN token from env
+
+const rpm = new RepositoryProjectsManager({
+  owner: 'babbel-sandbox',
+  repository: 'test-repo-jsaito-3',
```

Comment:
```suggestion
  repository: 'example',
```

(also in other places)